src/main/java/net/minecraftforge/fml/client/GuiSlotModList.java

```diff
@@ -86,20 +93,24 @@ protected void drawSlot(int idx, int right, int top, int height, Tessellator tes
         if (Loader.instance().getModState(mc) == ModState.DISABLED)
         {
             font.drawString(font.trimStringToWidth(name,       listWidth - 10), this.left + 3 , top +  2, 0xFF2222);
-            font.drawString(font.trimStringToWidth(version,    listWidth - 10), this.left + 3 , top + 12, 0xFF2222);
+            font.drawString(font.trimStringToWidth(version,    listWidth - (10 + 8 + 17)), this.left + 3 , top + 12, 0xFF2222);
             font.drawString(font.trimStringToWidth("DISABLED", listWidth - 10), this.left + 3 , top + 22, 0xFF2222);
```

Comment:
I will draw a picture later to explain what I thought. If I remember what I thought :P

> Am 22.03.2016 um 07:12 schrieb LexManos <notifications@github.com>:
> 
> In src/main/java/net/minecraftforge/fml/client/GuiSlotModList.java:
> 
> > @@ -86,20 +93,24 @@ protected void drawSlot(int idx, int right, int top, int height, Tessellator tes
> >          if (Loader.instance().getModState(mc) == ModState.DISABLED)
> >          {
> >              font.drawString(font.trimStringToWidth(name,       listWidth - 10), this.left + 3 , top +  2, 0xFF2222);
> > -            font.drawString(font.trimStringToWidth(version,    listWidth - 10), this.left + 3 , top + 12, 0xFF2222);
> > +            font.drawString(font.trimStringToWidth(version,    listWidth - (10 + 8 + 17)), this.left + 3 , top + 12, 0xFF2222);
> >              font.drawString(font.trimStringToWidth("DISABLED", listWidth - 10), this.left + 3 , top + 22, 0xFF2222);
> This number seems arbitrary.
> We use a 5 pixel border/buffer that's why the width is 10 {both sides}
> yours should be something like:
> 5 /left border/ + 5 /right border/ + image_width + 5 /buffer between image and edge*/
> 17 and 8 just sound arbitrary so i'm asking you to explain yourself.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
>